rhel7/profiles/stig-rhel7-disa.profile

```diff
@@ -119,7 +119,7 @@ selections:
     - selinux_policytype
     - disable_ctrlaltdel_reboot
     - accounts_umask_etc_login_defs
-    - installed_OS_is_certified
+    - installed_OS_is_vendor_supported
```

Comment:
Maybe `stig-rhel7-disa` should select both rule,` vendor_supported` and `FIPS_certified`.
@shawndwells would appreciate your input here